src/mn_setup.c

```diff
@@ -2362,6 +2362,9 @@ static setup_menu_t gen_settings2[] = {
      {"midi_player_menu"}, .strings_id = str_midi_player,
      .action = SetMidiPlayer},
 
+    {"Number of OPL Chips", S_THERMO, CNTR_X, M_THRM_SPC, {"num_opl_chips"},
+     .action = SetMidiPlayer},
```

Comment:
I'd prefer to keep this setting as config-only. If the performance impact isn't too great, a reasonable default that only enthusiasts would ever want to change seems like the way to go. In the future, maybe there will be a separate advanced MIDI menu, and options like this can be reconsidered at that time.

Reply:
I had some difficulty deciphering how the config system works. What exactly is required for a config-only option? Just the Bind_X call?

Reply:
A `BIND_NUM` call should work. It's range checked automatically by the min/max values you set. Maybe @rfomin can give some additional tips.

Reply:
Yes, just `BIN_NUM` macro will do it.

> there will be a separate advanced MIDI menu

I like the idea!